packages/entities/entities-shared/src/components/entity-empty-state/EntityEmptyState.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+  <div class="kong-ui-public-entity-empty-state">
+    <div
+      v-if="$slots.icon"
+      class="entity-empty-state-icon"
+    >
+      <slot name="icon" />
+    </div>
+    <div
+      v-else
+      class="entity-empty-state-image"
+    >
+      <slot name="image" />
+    </div>
+    <div class="entity-empty-state-content">
+      <div
+        v-if="title || $slots.title"
+        class="entity-empty-state-title"
+      >
+        <div :title="title">
+          {{ title }}
+        </div>
+        <span v-if="$slots['title-after']">
+          <slot name="title-after" />
+        </span>
+      </div>
+      <div
+        v-if="description || $slots.default"
+        class="entity-empty-state-description"
+      >
+        <slot name="default">
+          <p>
+            {{ description }}
+          </p>
+        </slot>
+      </div>
+      <div
+        v-if="pricing"
+        class="entity-empty-state-pricing"
+      >
+        <p>
+          <b>{{ t('emptyState.pricingTitle') }}</b> {{ pricing }}
+        </p>
+      </div>
```

Comment:
I just realized we need the ability to slot a link into this content. Can you have something like this instead?

```suggestion
      <div
        v-if="pricing"
        class="entity-empty-state-pricing"
      >
        <p>
          <b>{{ t('emptyState.pricingTitle') }}</b> <slot name="pricing">{{ pricing }}</slot>
        </p>
      </div>
```

Reply:
Good call out!